src/models/interfaces.ts

```diff
@@ -26,13 +26,6 @@ export enum ContentType {
     LocalizedTexts = 12,
 }
 
-export interface ISlickRange {
```

Comment:
are these removed because this doesn't yet support exporting by selection?

Reply:
No, there were three of these same interface in the file so I just removed two of them lol